lib/exporters/jhipster_application_exporter.js

```diff
@@ -49,10 +49,12 @@ function exportApplications(applications) {
  * @param application, the JDL application to exporters.
  * @return the exported application in its final form.
  */
-function exportApplication(application) {
+function exportApplication(application, configuration = {}) {
   checkForErrors(application);
   const exportableApplication = setUpApplicationStructure(application);
-  writeConfigFile(exportableApplication);
+  if (!configuration.skipYoRcGeneration) {
```

Comment:
can you add the `configuration` & `skipYoRcGeneration` attributes to the doc please?

Reply:
I was testing back there, forgot to add when I create the PR.
Added to JSDoc, but used a new format, can use a old format if you want.

Reply:
Nop that's good!